sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -1022,4 +1024,18 @@ public void closeActiveWindow() {
           "Cannot close active window and switch to parent window because only one is available!");
     }
   }
+
+  public boolean isFileExists(Path file_path) {
+    boolean file_exists;
+    return Files.exists(file_path);
+  }
+
+  public void waitForFileExists(Path file_path, int seconds) {
```

Comment:
check in the framework, we already have directly assert for this, we don't need to create it here